src/Reference_One.php

```diff
@@ -61,6 +61,15 @@ class Reference_One extends Reference
     public $read_only = false;
 
     /**
+     * Defines a label to go along with this field. Use getCaption() which
+     * will always return meaningfull label (even if caption is null). Set
+     * this property to any string.
+     *
+     * @var string
+     */
+    public $caption = null;
```

Comment:
No. We extend Reference which is separate class from Field. That's why we have to redefine these properties here.